docs/installation/linux/ubuntulinux.md

```diff
@@ -62,7 +62,7 @@ packages from the new repository:
 
 4. Add the new `GPG` key.
 
-        $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+        $ sudo apt-key adv --keyserver hkp://pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
 
```

Comment:
yes, I remembered that discussion, and actually I was browsing back the history of `install.sh`, because at some point it was changed to `ha.pool..`. Should we change it back to p80 in the install script?